ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -10,9 +10,4 @@ def layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=
     return ivy.layer_norm(x, normalized_shape, weight, bias, epsilon)
 
 
-@to_ivy_arrays_and_back
-@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
-def normalize(x, p=2, axis=1, epsilon=1e-12, name=None):
-    if axis < 0:
-        axis = ivy.get_num_dims(x) + axis
-    return ivy.lp_normalize(x, p=p, axis=axis)
+Removed conflicts.
```

Comment:
Please don't remove any code snippet unrelated to your changes.

Reply:
Hi, Oh I so sorry about I didn't close attention. Thank you.